final/NA/KBDL/KBDL.txt

```diff
@@ -0,0 +1,1918 @@
+	N42°28.08', W72°19.35'
+	N42°25.69', W72°19.22'
+	N42°25.01', W72°19.44'
+	N42°21.20', W72°19.19'
+	N42°20.17', W72°19.96'
+	N42°19.35', W72°20.15'
+	N42°18.93', W72°21.17'
+	N42°19.48', W72°21.23'
+	N42°20.90', W72°20.89'
+	N42°24.25', W72°21.88'
+	N42°27.54', W72°23.15'
+
+
+#Wachusett Reservoir
+line5 =
+	N42°24.19', W71°41.45'
+	N42°23.46', W71°41.63'
+	N42°22.98', W71°43.42'
+	N42°21.95', W71°44.01'
+	N42°21.61', W71°45.06'
+	N42°20.63', W71°45.00'
+	N42°20.72', W71°45.62'
+	N42°21.50', W71°45.59'
+	N42°22.14', W71°46.67'
+	N42°22.46', W71°46.54'
+	N42°22.25', W71°45.31'
+	N42°23.00', W71°44.38'
+	N42°23.30', W71°43.70'
+	N42°23.62', W71°44.04'
+	N42°24.05', W71°43.98'
+	N42°24.40', W71°42.53'
+	N42°24.19', W71°41.45'
+
+
+
+
+##############################################################
+[airport1]
+##############################################################
+
+name = Bradley International (Windsor Locks) Airport, bradley	; name of the main airport, <pronunciation>
+code = KBDL			; four letter code, displayed in airport menu (required; remove the #)
+
+runways = 
+	BDL_06_24,	6,	N41°55.92',	W72°41.79',	44.4,	9510,	0,	0,	173,	3,	44.4,	3,	224.4,	JANID,	8.0,	SURLE,	10.5,	120.3,	bradley tower
+	BDL_15_33,	15,	N41°56.54',	W72°41.60',	134.4,	6847,	0,	0,	168.8,	3.6,	134.4,	3,	314.4,	BREEN,	6.0,	HOMEY,	5.0,	120.3,	bradley tower
+	BDL_24_LAHSO,	24,	N41°57.04',	W72°40.33',	224.4,	5700,	0,	0,	160.9,	3,	224.4,	3,	44.4,	SURLE,	10.5,	Error,	0,	120.3,	bradley tower
+	BDL_33_LAHSO,	33,	N41°55.76',	W72°40.52',	314.4,	4400,	0,	0,	168.5,	3,	314.4,	3,	134.4,	HOMEY,	5.0,	Error,	0,	120.3,	bradley tower
+
+
+## identifier, name, position, true_heading, <length>, <displaced>, <displaced2>, <elevation>, <glideslope>, <localizer>, <glideslope2>, <localizer2>, <beaconname, distance>, <beaconname2, distance2>, <tower frequency>, <tower pronunciation>
+# identifier is a unique name used to define a runway usage configuration later in this file
+# name is one or two digits, optionally suffixed with an L, R or C character
+# position is the location of the beginning of the runway (including displaced threshold)
+# length is in feet
+# displaced threshold length in feet, displaced2 is for the other end of the runway
+# elevation is in feet, and must be within 1000 feet difference from the airspace ground elevation defined above
+# glideslope is the angle in degrees; the default value is 3.0
+# localizer is the true heading in degrees of the ILS until touchdown; by default it's the same as runway true heading, but a small offset is allowed
+# beacon is an optional point on the localizer at a certain distance (NM) from the (displaced) threshold
+# second beacon is a point on the localizer when runway is used in opposite direction (rev)
+# tower frequency is used for handoff readbacks
+
+climbaltitude = 7000 		; maximum initial climb altitude in feet for departures
+
+sids = 
+	veers,	N41.54.37.120,	W073.17.11.760,	veers
+	bowan,	N42.06.23.140,	W073.31.28.950,	bowan
```

Comment:
I'm not aware of BOWAN being a departure gate -- MOBBS is on the same airway, though.

Reply:
That was based on aircraft being able to be cleared direct to bowan without coordination, but I'll switch that to mobbs